StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -177,7 +177,8 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
     private func buildPaymentMethodRows(paymentMethods: [STPPaymentMethod]) -> [SavedPaymentMethodRowButton] {
         return paymentMethods.map { paymentMethod in
             let button = SavedPaymentMethodRowButton(paymentMethod: paymentMethod,
-                                                     appearance: configuration.appearance)
+                                                     appearance: configuration.appearance,
+                                                     showDefaultPMBadge: configuration.allowsSetAsDefaultPM && paymentMethod.stripeId == elementsSession.customer?.defaultPaymentMethod)
```

Comment:
This feels repetitive:
```
configuration.allowsSetAsDefaultPM && paymentMethod.stripeId == elementsSession.customer?.defaultPaymentMethod)
```

How about we create a new function:
```
    func isDefaultPaymentMethod(paymentMethodId: String) -> Bool {
        return configuration.allowsSetAsDefaultPM && paymentMethodId == elementsSession.customer?.defaultPaymentMethod
    }

```

Then, calling it looks like:
```
 let button = SavedPaymentMethodRowButton(paymentMethod: paymentMethod,
                                                     appearance: configuration.appearance,
                                                     showDefaultPMBadge: isDefaultPaymentMethod(paymentMethodId: paymentMethod.stripeId))

....
let newButton = SavedPaymentMethodRowButton(paymentMethod: updatedPaymentMethod,
                                                    appearance: configuration.appearance,
                                                    showDefaultPMBadge: isDefaultPaymentMethod(paymentMethodId: updatedPaymentMethod.stripeId))
```